swagger.yaml

```
@@ -1366,6 +1570,50 @@ components:
         - IN_PROGRESS
         - DONE
 
+    ChangeProposal:
+      type: object
+      description: Change proposal
+      required:
+        - status
+        - taskId
```

Should reason also be required?
```suggestion
      required:
        - status
        - taskId
        - reason
```